installer/pkg/workflow/install.go

```diff
@@ -19,8 +19,8 @@ func InstallFullWorkflow(clusterDir string) Workflow {
 			readClusterConfigStep,
 			generateTLSConfigStep,
 			generateClusterConfigMaps,
-			installAssetsStep,
 			generateIgnConfigStep,
+			installAssetsStep,
```

Comment:
Maybe I'm missing something obvious, but I don't understand why we need to `generateIgnConfigStep` before `installAssetsStep`.  Can you drop some notes here and/or in the commit message?

Reply:
hey @wking for aws a secret is generated with the ignition config https://github.com/openshift/installer/pull/119/commits/79585d1cee5ecfe39030e8a30127ce1908da96cc#diff-165e895590d153f6ae0c9c071c366da9 which is then consumed by the actuator, hence to generate the secret manifest asset we need the ign config file to exist. This will eventually along with the operator config should go into some kind of interface for operators

Reply:
I thought the IgnConfig step required assets though (specifically the root CA).